README.md

```diff
@@ -51,3 +51,7 @@ gulp
       * fetch.user.report.error - logged when encountered problems during search for user reports
       * fetch.user.error - logged when encountered error during user fetch
 
+### Status
+
+  * `Settings are empty` - `error` - returned when we have no script code saved in the settings
+  * `Settings are referencing invalid values` - `error` - returned when we have a script with syntax error
```

Comment:
If we can't do static analysis on the code then we can just check that it's not empty. Otherwise we need to use the fixtures to pass user, events, changes, accounts, segments etc...